src/arch/timerHandlerFactory.h.in

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
Please no code before the copyright header.

Reply:
done